Controllers/APIs/EmployeesController.cs

```diff
@@ -132,22 +249,61 @@ public async Task<ActionResult> RefreshEmployeeStatus()
             try
             {
                 // Update existing employee statuses.
-                await EmployeeReconciler.UpdateEmployeeStatuses();
+                var taskResult = await employeeReconciler.UpdateEmployeeStatusesAndLog();
 
-                await logger.LogSuccess(
+                emailService.SendTaskResultEmail(taskResult);
+
+                return Ok();
+            }
+            catch (Exception e)
+            {
+                return await ApiResponseHelper.LogFailureAndSendStacktrace(
+                    this,
                     TaskEnum.RefreshStatuses,
-                    $"Manually-triggered refresh of employee statuses."
+                    e,
+                    logger
+                );
+            }
+        }
+
+        [AllowAnonymous]
```

Comment:
Is this required as the cronjob doesn't have a keycloak credential?

Reply:
Yes, that's right.